config/features/deprecated_flags.go

```diff
@@ -57,6 +59,20 @@ var (
 		Usage:  deprecatedUsage,
 		Hidden: true,
 	}
+
+	deprecatedBeaconRPCGatewayProviderFlag = &cli.StringFlag{
+		Name:   "beacon-rpc-gateway-provider",
+		Usage:  deprecatedUsage,
+		Value:  "127.0.0.1:3500",
```

Comment:
What's the point of having `Value` in a deprecated flag?

Reply:
i guess no real point ...